.github/workflows/publish-webapp.yml

```diff
@@ -12,15 +12,15 @@ on:
     branches: [ "main" ]
     # Publish semver tags as releases.
     tags: [ 'v*.*.*' ]
-  pull_request:
-    branches: [ "main" ]
+  # pull_request:
```

Comment:
Will flip this back on once we stabilize a bit. No need to keep running it over and over right now